app/controllers/reviews_controller.rb

```diff
@@ -85,6 +88,28 @@ def user_params
       params.require(:review).permit(:name, :email)
     end
 
+    def delivery_params
+      params.require(:review).permit(:maxDevTime)
+    end
+
+
+    def update_maxDevTime(restaurant_id)
+      review_maxDevTime = Integer(delivery_params["maxDevTime"])
```

Comment:
why do you do casting?

Reply:
Agree with @yossis, no need for casting since the model validation will take care of it

Reply:
without the casting i have a problem with the comparison.
the max_delivery_time comes from the form as a string